### PR TITLE
UI for the delete contact modal

### PR DESCRIPTION
**Pull request by jascky**
## Description
Created the DeleteModal component, adding it's template end event emitter for deleting a contact 

## Fixes #167799225
https://www.pivotaltracker.com/n/projects/2382820/stories/167799225

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: ui/web/src/components/DeleteModal.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<template>
+    <div>
+      <v-row justify="center">
+
```
Comment (MiguelDiazArkus) [inline ui/web/src/components/DeleteModal.vue]: Remove empty lines in template code.

**Review by MiguelDiazArkus — CHANGES_REQUESTED** [review]
@jascky Please remove empty lines in template code.

File: ui/web/src/components/DeleteModal.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+<script>
+export default {
+    data(){
+        return {
+            
+        }
+    },
+    props:{
+        dialog: Boolean,
```
Comment (maumaigre) [inline ui/web/src/components/DeleteModal.vue]: showDialog*

**Review by maumaigre — CHANGES_REQUESTED** [review]

File: ui/web/src/components/DeleteModal.vue (hunk 1 above)
Comment (jascky) [reply ui/web/src/components/DeleteModal.vue]: fixed

File: ui/web/src/components/DeleteModal.vue (hunk 2 above)
Comment (jascky) [reply ui/web/src/components/DeleteModal.vue]: fixed

**Review by CSaavedra96 — APPROVED** [review]
Looks good

**Review by maumaigre — APPROVED** [review]

**Review by MiguelDiazArkus — APPROVED** [review]
looks good

**Pull request merged**